### ISSUE TITLE
[Bug Report] Undue UserWarning: We recommend you to use a symmetric and normalized Box action space (range=[-1, 1])

### ISSUE THREAD

**Issue by qgallouedec** [openai/gym#2885]
It seems like there is an inversion in the `gym.utils.env_checker.check_env` function: observation space is checked with `check_action_space`

https://github.com/openai/gym/blob/053ee809324861dc1bec19feee97b47ba1557a94/gym/utils/env_checker.py#L237

and action space is checked with `check_observation_space`

https://github.com/openai/gym/blob/053ee809324861dc1bec19feee97b47ba1557a94/gym/utils/env_checker.py#L233

causing undue warnings such as 

```log
UserWarning: WARN: We recommend you to use a symmetric and normalized Box action space (range=[-1, 1])
```

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/openai/gym/issues) in the repo (**required**)

**Comment by pseudo-rnd-thoughts** [comment]
Wow, thanks for spotting that. That is impressive that I messed that up. As I have done a number of upgrades / bug fixes to the environment checker, then I have added it to that PR https://github.com/openai/gym/pull/2873

**Comment by pseudo-rnd-thoughts** [comment]
@qgallouedec Would you be able to close your PR for this #2886 and clone my PR to check that it runs against your environments to make sure it doesn't raise any incorrect error?

**Comment by qgallouedec** [comment]
Fixed in cdeedd8e3ba454c48af2760bc26b70866e2d7141
